blocks/components/editable/style.scss

```diff
@@ -10,4 +10,15 @@
 	&:focus {
 		outline: none;
 	}
+
+	&.is-empty {
+		position: relative;
+	}
+
+	&.is-empty:before {
```

Comment:
@BE-Webdesign Huh, I wasn't aware of that pseudo-element. You're probably right on the point about browser support. Out of curiosity, for those browsers that do support it, would it work on non-input types like how we're using it here?